### PR TITLE
Make comment box shrink on submit

### LINKED ISSUE #13298: Comment box not shrinking on submit
**Describe the bug**
Comment boxes grow as you type but when you press submit they stay the size they were when they had content.

**To Reproduce**
- Go to a post
- Write a long comment
- Post the comment
- See the comment box is still large.

**Expected behaviour**
I'd expect the comment box the shrink back down to it's original size.

**Screenshots**
image

**Desktop (please complete the following information):**

- OS, version: Window 10
- Browser, version: Chrome 89.0.4389.114

**Additional context**
Another small bug that doesn't really cause any harm.

### PR DESCRIPTION
**Pull request by oyeyipowale**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
The main comment box on the articles detail page does not reset to default height after successful submission. The inline style height set on the textarea element is not being removed.

## Related Tickets & Documents
Fixes #13298

## Added tests?

- [ ] Yes
- [X] No, and this is why: this is a UI bug fix
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

Thinking Bob

### CODE REVIEW

File: app/assets/javascripts/initializers/initializeCommentsPage.js.erb [hunk 1]
```diff
@@ -267,6 +267,8 @@ function handleCommentSubmit(event) {
               input.value = newComment.comment_template || "";
             });
 
+            commentInputs[1].style.height = null
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeCommentsPage.js.erb:270]: tiny change: could you add the trailing `;`?

**Review by rhymes — COMMENTED** [review]
Great catch @oyeyipowale!

I tested it and it works! I added a small request :)

File: app/assets/javascripts/initializers/initializeCommentsPage.js.erb (hunk 1 above)
Comment (oyeyipowale) [reply app/assets/javascripts/initializers/initializeCommentsPage.js.erb:270]: Missed that thanks

**Review by Farhaduneci — COMMENTED** [review]
Test

File: app/assets/javascripts/initializers/initializeCommentsPage.js.erb [hunk 2]
```diff
@@ -267,6 +267,8 @@ function handleCommentSubmit(event) {
               input.value = newComment.comment_template || "";
             });
 
+            commentInputs[1].style.height = null;
```
[the remark this answers is not in the document]
Comment (oyeyipowale) [reply app/assets/javascripts/initializers/initializeCommentsPage.js.erb:270]: okay, good thought.

**Review by rhymes — APPROVED** [review]

**Comment by aitchiss** [comment]
Thanks for the fix @oyeyipowale! ✨

**Comment by cmgorton** [comment]
@oyeyipowale would you mind making a quick post on DEV using this [template](https://dev.to/new/devbugsmash). That way we can make sure you get the Bug Smash badge and stickers 😄

**Pull request merged**